homeassistant/components/screenlogic/const.py

```diff
@@ -13,4 +13,6 @@
     slugify(name): num for num, name in COLOR_MODE.NAME_FOR_NUM.items()
 }
 
+LIGHT_CIRCUITS = {CIRCUIT_FUNCTION.INTELLIBRITE, CIRCUIT_FUNCTION.LIGHT}
```

Comment:
Suggest changing to `LIGHT_CIRCUIT_FUNCTIONS` to avoid confusion on contents - function names vs references to the actual circuits.